packages/salesforcedx-apex-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -252,9 +252,9 @@ export class ApexReplayDebug extends LoggingDebugSession {
     // If the projectPath isn't set then don't bother with heap dump processing
     if (this.projectPath && this.logContext.scanLogForHeapDumpLines()) {
       if (
-        !await this.logContext.fetchOverlayResultsForApexHeapDumps(
+        !(await this.logContext.fetchOverlayResultsForApexHeapDumps(
```

Comment:
Does this actually affect the waiting for the heap dumps to process or was this something tagged by formatting?

Reply:
Auto-formatting